content/kapp-controller/_index.html

```diff
@@ -55,108 +64,120 @@ <h2>Basic Usage</h2>
   serviceAccountName: default-ns-sa
   fetch:
   - git:
-      url: https://github.com/vmware-tanzu/carvel-simple-app-on-kubernetes
+      url: https://github.com/k14s/k8s-simple-app-example
       ref: origin/develop
-      subPath: config-step-1-minimal
+      subPath: config-step-2-template
   template:
   - ytt: {}
   deploy:
   - kapp: {}
-{{< / highlight >}}
-        <p>Let's apply configuration to your cluster and see it get reconciled.</p>
-        {{< highlight bash "" >}}
-$ kubect apply -f simple-app.yml
+{{< / highlight>}}
+    <p>Apply the App CR to your cluster and verify the deployment was successful:</p>
+{{< highlight bash "">}}
+$ kubectl get app simple-app
```

Comment:
dropped the apply command

Reply:
It was intentional since these example aren't meant to be actually run, but I was worried it would be confusing so I can add back